cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -101,83 +101,83 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<functions-radians, radians()>>               | Trigonometric     | Converts degrees to radians.
-|<<functions-rand, rand()>>                     | Numeric           | Returns a random number in the range from 0 (inclusive) to 1 (exclusive), `[0, 1)`.
-|<<functions-range,range()>>                    | List              | Returns numerical values in a range.
-|<<functions-reduce,reduce()>>                  | List              | Runs an expression against individual elements of a list, storing the result of the expression in an accumulator.
-|<<functions-relationships,relationships()/rels()>>    | List              | Returns all relationships in a path.
-|<<functions-replace,replace()>>                | String            | Returns a string with the search string replaced by the replace string, replacing all occurrences.
-|<<functions-reverse,reverse()>>                | String            | Returns the original string reversed.
-|<<functions-right,right()>>                    | String            | Returns a string containing the right n characters of the original string.
-|<<functions-round, round()>>                   | Numeric           | Returns the numerical expression, rounded to the nearest integer.
-|<<functions-rtrim,rTrim()>>                    | String            | Returns the original string with whitespace removed from the right side.
-|<<functions-sign, sign()>>                     | Numeric           | Returns the signum of a number -- zero if the expression is zero, `-1` for any negative number, and `1` for any positive number.
-|<<functions-sin, sin()>>                       | Trigonometric     | Returns the sine of the expression.
-|<<functions-single,single()>>                  | Predicate         | Returns true if the predicate holds for exactly one of the elements in the list.
+|<<functions-rand, rand()>>                     | Numeric           | Returns a random floating point number in the range from 0 (inclusive) to 1 (exclusive); i.e. [0, 1).
+|<<functions-range,range()>>                    | List              | Returns a list comprising all integral values within a range bounded by a start and end value, where the difference between any two consecutive values is constant.
+|<<functions-reduce,reduce()>>                  | List              | Returns the value resulting from the application of an expression on each successive element in a list in conjunction with the result of the computation thus far.
+|<<functions-relationships,relationships()/rels()>>    | List              | Returns a list containing all the relationships in a path.
+|<<functions-replace,replace()>>                | String            | Returns a string in which all occurrences of a specified string in the original string have been replaced by another string.
+|<<functions-reverse,reverse()>>                | String            | Returns a string in which all characters in the original string have been reversed.
```

Comment:
Except you do not reverse the characters, but the order of the characters, or in other words reverse the 'string', which the original description said better, I thought.